Cargo.toml

```diff
@@ -34,6 +34,7 @@ members = [
     "roles/v2/pool",
     "roles/v2/test-utils/mining-device",
     "roles/v2/test-utils/pool",
+    "utils/anyhow",
```

Comment:
i am hesitant to include a new way to handle errors in this PR.  i think it will be confusing if part of the library uses error enums in various `error.rs` files, but then sometimes uses this custom `anyhow` impl. can we just handle errors like the rest of the lib does it in this PR?